test/cookbooks/test-module/recipes/default.rb

```diff
@@ -17,4 +17,9 @@
 # limitations under the License.
 #
 
+# delete firewire-core module to test 'check_availability' attribute
+file "/lib/modules/#{node['kernel']['release']}/kernel/drivers/firewire/firewire-core.ko" do
+  action :delete
+end
+
```

Comment:
Since you are using the default recipe you shouldn't create a suite, the inspec should be enough in the default suite.